index.html

```diff
@@ -0,0 +1,197 @@
+        <p class="text-light fw-semibold">@Colosseum Ohara West Blue</p>
+      </div>
+    </header>
+    <main>
+      <section id="program" class="d-flex flex-column align-items-center pb-3">
+        <h2 class="text-center fw-bolder pt-5 text-light">Main Program</h2>
+        <span class="line-primary"></span>
+        <div
+          class="d-flex flex-column flex-md-row flex-wrap justify-content-md-center m-3"
+        >
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-masks-theater fs-1 p-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Perfomances</h3>
+            <p class="p-1">
+              A lot of performers will join us at that time to blow our mind.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-pen-to-square fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Autographs</h3>
+            <p class="p-1">
+              All your favorite authors will gather to make your dream a
+              reality.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-solid fa-handshake fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">
+              Cosplayer meet&greet
+            </h3>
+            <p class="p-1">
+              Every big cosplayers from all over the world will be there.
+            </p>
+          </div>
+          <div
+            class="d-flex flex-md-column align-items-center bg-light opacity-75 rounded-1 m-3 p-3 col-md-2 program-item"
+          >
+            <i class="fa-sharp fa-solid fa-star fs-1"></i>
+            <h3 class="fw-semibold m-2 program-title fs-6">Special events</h3>
+            <p class="p-1">
+              The "gear fourth" show will be the wild card but more special
+              events will also take place.
+            </p>
+          </div>
+        </div>
+        <button type="button"
+          class="btn btn-danger p-3 rounded-0 text-white fw-bold mx-auto text-uppercase"
+        >
+          Claim your badge
+        </button>
+      </section>
+      <section id="guest" class="d-flex flex-column align-items-center pb-3">
+        <h2 class="text-center fw-bolder pt-5 mb-3">Our virtual guests</h2>
+        <span class="line-primary"></span>
+        <ul class="row mt-5 list-unstyled p-3 justify-content-md-between"></ul>
+        <button type="button"
+          class="btn btn-danger p-3 rounded-0 text-white fw-bold mx-auto text-uppercase d-md-none more-or-less"
+        >
+          Load more
+        </button>
```

Comment:
Kindly note that the mobile version of the home page ends at the more button, hence does not have the partners and the footer part as shown below. 
Also check the design and color of the more button and fix it since there are some disparities as shown below :point_down: 

Your design 

![Screenshot from 2023-01-12 08-18-02](https://user-images.githubusercontent.com/65233860/212013859-334704a6-6dc9-48f2-b545-ad402e238f5f.png)


Actual design

![Screenshot from 2023-01-12 08-13-25](https://user-images.githubusercontent.com/65233860/212013841-0ae9db27-d38b-4bf6-8dde-7801ebb05094.png)